tethysext/atcore/models/resource_workflow_steps/table_input_rws.py

```diff
@@ -46,7 +47,8 @@ def default_options(self):
             'plot_columns': [],
             'optional_columns': [],
             'max_rows': self.DEFAULT_MAX_ROWS,
-            'empty_rows': self.DEFAULT_EMPTY_ROWS
+            'empty_rows': self.DEFAULT_EMPTY_ROWS,
+            'fixed_rows': self.DEFAULT_FIXED_ROWS
```

Comment:
Please add explanation of the `fixed_rows` option in the doc string for the step class (after line 23).